api/src/task/notifications/search_notification.py

```diff
@@ -101,6 +166,76 @@ def collect_email_notifications(self) -> list[UserEmailNotification]:
 
         return users_email_notifications
 
+    def _build_notification_message(self, opportunities: Sequence[Opportunity]) -> str:
+
+        # Build message intro based on number of opportunities
+        if len(opportunities) == 1:
+            message = (
+                "A funding opportunity matching your saved search query was recently published.\n\n"
+            )
+        else:
+            message = "The following funding opportunities matching your saved search queries were recently published.\n\n"
+
+        # Add details for each opportunity
+        for opportunity in opportunities:
+            summary = (
+                opportunity.current_opportunity_summary.opportunity_summary
+                if opportunity.current_opportunity_summary
+                else None
+            )
+
+            if not summary:
+                continue
+
+            # Add opportunity title (empty line before title)
+            message += f"{opportunity.opportunity_title}\n\n"
+
+            # Add status
+            status = (
+                str(opportunity.opportunity_status).capitalize()
+                if opportunity.opportunity_status
+                else "Unknown"
+            )
+            message += f"Status: {status}\n"
+
+            # Add submission period
+            if summary.is_forecast:
+                message += "Submission period: To be announced.\n"
```

Comment:
If it's a forecast, the `forecasted_post_date` and `forecasted_close_date` may be set and could tell this info. Don't need to handle necessarily in this PR.